readers/api/transport.go

```diff
@@ -54,7 +54,7 @@ const (
 var errUserAccess = errors.New("user has no permission")
 
 // MakeHandler returns a HTTP handler for API endpoints.
-func MakeHandler(svc readers.MessageRepository, uauth magistrala.AuthServiceClient, taauth magistrala.AuthzServiceClient, svcName, instanceID string) http.Handler {
+func MakeHandler(svc readers.MessageRepository, uauth magistrala.AuthServiceClient, taauth magistrala.ThingsServiceClient, svcName, instanceID string) http.Handler {
 	opts := []kithttp.ServerOption{
```

Comment:
We can have some thing like below 
```suggestion
func MakeHandler(svc readers.MessageRepository, auth magistrala.AuthServiceClient, things magistrala.ThingsServiceClient, svcName, instanceID string) http.Handler {
```